spec/02-integration/04-admin_api/04-plugins_routes_spec.lua

```diff
@@ -1,187 +1,230 @@
 local helpers = require "spec.helpers"
-local cjson = require "cjson"
+local cjson   = require "cjson"
 
-describe("Admin API", function()
-  local client
-  setup(function()
-    assert(helpers.dao:run_migrations())
-    assert(helpers.db:truncate())
+for _, strategy in helpers.each_strategy() do
 
-    assert(helpers.start_kong())
-    client = helpers.admin_client()
-  end)
-  teardown(function()
-    if client then client:close() end
-    helpers.stop_kong()
-  end)
+  describe("Admin API #" .. strategy, function()
+    local dao
+    local db
+    local client
+
+    setup(function()
+      _, db, dao = helpers.get_db_utils(strategy)
+      assert(helpers.start_kong({
+        database = strategy
+      }))
+
+      dao:run_migrations()
 
-  describe("/plugins/enabled", function()
-    it("returns a list of enabled plugins on this node", function()
-      local res = assert(client:send {
-        method = "GET",
-        path = "/plugins/enabled",
-      })
-      local body = assert.res_status(200, res)
-      local json = cjson.decode(body)
-      assert.is_table(json.enabled_plugins)
-      assert.True(#json.enabled_plugins > 0)
+      dao:truncate_tables()
+      db:truncate()
```

Comment:
These operations are now handled by `get_db_utils()`. Will update locally but good for the next time :)